solvers/pgd.py

```diff
@@ -0,0 +1,80 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy import sparse
+    from numba import njit
```

Comment:
you don't need to vectorize operations here (the cost of iteration is dominated by X.T @ R), so numba and jit are not needed